src/inspector/node_protocol.pdl

```diff
@@ -71,6 +71,11 @@ experimental domain NodeWorker
   # Detaches from all running workers and disables attaching to new workers as they are started.
   command disable
 
+  # Detached from worker with given sessionId.
+  command detach
```

Comment:
Do we need `attach` for symmetry? The problem I see is that we do not track worker separately from a session so introducing attach would definitely make it more confusing...

Reply:
In this case should we have a little bit crazy `NodeRuntime.disconnect` method that works for main and worker thread?
I don't know any use case for `NodeWorker.attach`. Since workers are not waiting for attach by default, this method does not add any value without `waitForDebuggerOnStart` flag. Could we add it later when we know use case better?

Reply:
One potential problem I see with the `detach` is that the worker is kind of "lost" once session is detached. A frontend may detach from the worker early and then there will be no way of attaching again. IMHO, `attach` can be implemented later if needed.

Reply:
> In this case should we have a little bit crazy NodeRuntime.disconnect method that works for main and worker thread?

Don't think this is necessary.

Reply:
Feel free to ping me when someone would ask about this feature!